packages/jest-fake-timers/src/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export {default as JestFakeTimers} from './jestFakeTimers';
```

Comment:
Not default export since I want to keep the door open for Lolex as an alternative, ref #7776